package.json

```diff
@@ -19,12 +18,14 @@
     "not dead"
   ],
   "scripts": {
-    "develop": "gatsby develop",
-    "dev": "gatsby develop",
-    "start": "gatsby develop",
+    "develop": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
+    "dev": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
+    "start": "ENABLE_GATSBY_REFRESH_ENDPOINT=true gatsby develop",
```

Comment:
This is three different ways to do the same thing. Can we just have `dev`?

Reply:
Sure 😂  Was thinking maybe any new devs come to the project is used to typing it one way or another, but totally not needed